### PR TITLE
fixed right panel to be sticky

### PR DESCRIPTION
**Pull request by Barylo**
Right panel now is sticky when scroll down the page.

### CODE REVIEW

File: src/components/RightPanel.tsx [hunk 1]
```diff
@@ -20,8 +26,10 @@ export const RightPanel = () => {
       <StickySearch>
         <AsyncSearch />
       </StickySearch>
-      {!location.pathname.includes("trends") && <Trends />}
-      <Recommendations />
+      <FixMenu>
+        {!location.pathname.includes("trends") && <Trends />}
+        <Recommendations />
+      </FixMenu>
     </Wrapper>
```
Comment (DavidKorystov) [inline src/components/RightPanel.tsx:28]: wrap asyncSearch also. Use one common div  for two components

**Review by DavidKorystov — CHANGES_REQUESTED** [review]

File: src/components/RightPanel.tsx [hunk 2]
```diff
@@ -7,21 +7,22 @@ import Trends from "../containers/Trends";
 
 const Wrapper = styled.div``;
```
Comment (DavidKorystov) [inline src/components/RightPanel.tsx:8]: remove this styled div, use default

**Review by DavidKorystov — CHANGES_REQUESTED** [review]

File: src/components/RightPanel.tsx [hunk 3]
```diff
@@ -12,6 +12,12 @@ const StickySearch = styled.div`
   top: 1.5%;
 `;
 
+const FixMenu = styled.div`
+  position: fixed;
```
Comment (KirillSerg) [inline src/components/RightPanel.tsx:11]: I hope it won't hurt to scroll the right panel when there are a lot of recommendations

**Review by KirillSerg — APPROVED** [review]

**Review by VitaliiDenysiuk — CHANGES_REQUESTED** [review]
I have not scroll and at resolution 1280:720 I cannot see "What to follow"
and such will  happen at all screen when will be a lot of  trends
image
at twitter there is scroll

File: src/components/RightPanel.tsx (hunk 2 above)
Comment (Barylo) [reply src/components/RightPanel.tsx:8]: when I remove it. the right panel covers the left panel

**Review by VitaliiDenysiuk — APPROVED** [review]
lgtm

File: src/components/RightPanel.tsx (hunk 2 above)
Comment (DavidKorystov) [reply src/components/RightPanel.tsx:8]: I don't understand how Wrapper(without styles) !== div.  Maybe you're just deleting this Wrapper?

Comment (Barylo) [reply src/components/RightPanel.tsx:8]: When I delete Wrapper the right panel slides to the left

**Review by alexandrtovmach — APPROVED** [review]

**Pull request merged**